ophirofox/content_scripts/config.js

```diff
@@ -98,6 +98,13 @@ const ophirofox_config_list = [
     AUTH_URL:
       "http://scd1.univ-fcomte.fr/login?url=https://nouveau.europresse.com/access/ip/default.aspx?un=FCOMTET_1",
   },
+  {
+    name: "Université de Pau et des Pays de l'Adour",
+    domains: ["univ-pau.fr"],
+    LOGIN_URL: "https://sso.univ-pau.fr/cas/login",
+    AUTH_URL:
+      "https://nouveau-europresse-com.rproxy.univ-pau.fr/access/ip/default.aspx?un=uppaT_2",
```

Comment:
Est-ce que vous êtes sûrs, pour ici ? Le format semble différent de toutes les autres universités. Il ne faudrait pas plutôt : 

```suggestion
      "https://rproxy.univ-pau.fr/login?url=https://nouveau.europresse.com/access/ip/default.aspx?un=uppaT_2",
```